src/Client/AzureManaged/DurableTaskSchedulerClientOptions.cs

```diff
@@ -173,4 +217,35 @@ this.Credential is not null
                     nameof(connectionString));
         }
     }
+
+    /// <summary>
+    /// Options used to configure retries used when making calls to the Scheduler.
+    /// </summary>
+    public class ClientRetryOptions
+    {
+        /// <summary>
+        /// Gets or sets the maximum number of times a call should be retried.
+        /// </summary>
+        public int? MaxRetries { get; set; }
+
+        /// <summary>
+        /// Gets or sets the initial backoff in milliseconds.
+        /// </summary>
+        public int? InitialBackoffMs { get; set; }
+
+        /// <summary>
+        /// Gets or sets the maximum backoff in milliseconds.
+        /// </summary>
+        public int? MaxBackoffMs { get; set; }
+
+        /// <summary>
+        /// Gets or sets the backoff multiplier for exponential backoff.
+        /// </summary>
+        public double? BackoffMultiplier { get; set; }
+
+        /// <summary>
+        /// Gets or sets the list of status codes that can be retried.
+        /// </summary>
+        public List<StatusCode>? RetryableStatusCodes { get; set; }
```

Comment:
Optional: Consider using an interface here, like `IList<StatusCode>?` or `ICollection<StatusCode>?`. This increases the number of list/collection types that customers can use to configure this property.